backend/app/bin/prestart-dev.sh

```diff
@@ -1,9 +1,10 @@
 #! /usr/bin/env bash
+# shellcheck source=SCRIPTDIR/live-reload.sh
```

Comment:
A bit pedantic, but I think I'd prefer if this comment line went above the `source` line executing `live-reload.sh`